src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -81,6 +81,11 @@ def test_native_app_create_scenario(self):
                        '5778995a-e1bf-45b8-affa-663a9f3f4d04')
         ]).get_output_in_json()
         self.kwargs['id'] = result['appId']
+        self.kwargs['cert_file'] = 'MIID5TCCAs2gAwIBAgIUZRHlYR+OJL2e4zwAPb8Q4kN+wQMwDQYJKoZIhvcNAQELBQAwgYExCzAJBgNVBAYTAmF1MQswCQYDVQQIDAJzaDELMAkGA1UEBwwCc2gxCzAJBgNVBAoMAm1zMQswCQYDVQQLDAJtczEXMBUGA1UEAwwOcWlhbndlbnRlc3RrZXkxJTAjBgkqhkiG9w0BCQEWFnFpYW53ZW50ZXN0a2V5QDE2My5jb20wHhcNMjAwNjAxMDY1ODAzWhcNMjEwNjAxMDY1ODAzWjCBgTELMAkGA1UEBhMCYXUxCzAJBgNVBAgMAnNoMQswCQYDVQQHDAJzaDELMAkGA1UECgwCbXMxCzAJBgNVBAsMAm1zMRcwFQYDVQQDDA5xaWFud2VudGVzdGtleTElMCMGCSqGSIb3DQEJARYWcWlhbndlbnRlc3RrZXlAMTYzLmNvbTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBANreNIRblW8dTuDZmRaS0AyhXsvmAe7C3BbrcYrffz0xmDgGqel5RXlfrwXRsghok1CygKUyO3KZYztShnnwin0oz23SUJcIkf9miXOBXnDJqNlFLgdVUHBbBU3gSv15UtIurl4V2Y4fi4gpHlMCAmXJGWMhHt/OlcAWWUvpLtGYiHIfzJZrXXBOJSNpcLLMsCJScSFNbmPsZv9ay1RSGLIF90X2ZiC0o8QjACULotQVfCH9Otg/FRXVWQGq6orOJLHp/+qcmMQKTxWPaSCoHGdMEh2N3gVc9syzH1qj64SwP2T4IGZdhEEHRoGDwfEm+tfk6C33qWfIh1d+ZqyIdT0CAwEAAaNTMFEwHQYDVR0OBBYEFDKjcHtP/v52XR6NjPdkZiGtjY4oMB8GA1UdIwQYMBaAFDKjcHtP/v52XR6NjPdkZiGtjY4oMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAEqAxq/fK5IKUA3wNrREG0A5LKx8kEFbLdhdqHjpwBXb4qGJiRg+ffJBTMYdq1Q34Wr01VB30Az7RxBPARteg7B8AiksyAw2NFhmyafuATQDzBPxNqTwAa0pqn1WJ9SfnWfxfFEXJhuvT7qNKoWstgfIGMtv17fzCKfVvk8WIVPnZSxUXxARwF9fUCIDDdT8YnkSyIYJQVGeowxMZw6G7A8HP+6tlwtq9V8M1T7BVBpnuc7lwvTzMCW1IUYIRZOTn1A3lNXFqctS/E8GF7pUtSYDOtMGFU1qqxSqLMNejLs82g07SXmuGG67CXIweULfMJgGXjSSdLOtG/QDH8pbWyo='
```

Comment:
Let's add the **command to generate this cert** here, in case in the future the cert expires and live run fails.